database/postgres/postgres_game.py

```diff
@@ -0,0 +1,102 @@
+from typing import Dict
+from datetime import datetime
+from database.models import Games
+from database.postgres.postgres_country import save_country
+from database.postgres.postgres_team import save_team
+from database.postgres.postgres_league import save_league
+
+def save_games(json_data: Dict, sport_id, session) -> None:
```

Comment:
json_data must be replaced with a model/class representing "games" list or something like that